Tests/runTestSimulation.m

```diff
@@ -75,22 +71,30 @@
 if ~isfolder(fullfile(pathRepo,'Debug','TestResults'))
     mkdir(fullfile(pathRepo,'Debug','TestResults'));
 end
-S.subject.save_folder  = fullfile(pathRepo,'Debug','TestResults'); 
+S.misc.save_folder  = fullfile(pathRepo,'Debug','TestResults'); 
 result_filename = [model_name '_test_' datestr(datetime,30)];
-S.post_process.result_filename = result_filename;
+S.misc.result_filename = result_filename;
 
 % update paths
-S.subject.IG_selection = replace(S.subject.IG_selection, S.misc.main_path, pathRepo);
+S.solver.IG_selection = replace(S.solver.IG_selection, S.misc.main_path, pathRepo);
 osim_path = replace(res_ref.model_info.osim_path, S.misc.main_path, pathRepo);
 
 S.misc.main_path = pathRepo;
 
 
-if ~isempty(options.CasADi_path)
-    S.solver.CasADi_path = options.CasADi_path;
-else
-    S.solver = rmfield(S.solver,'CasADi_path');
+% Make sure casadi path is set up correctly
+if isempty(options.CasADi_path)
+    try
+        S.solver.CasADi_path = casadi.GlobalOptions.getCasadiPath();
+    catch
+        error("Please add CasADi to the matlab search path, or pass the path " + ...
+            "to your CasADi installation (top folder) to 'CasADi_path'.")
```

Comment:
```suggestion
            "to your CasADi installation (top folder) to 'options.CasADi_path'.")
```

Reply:
The input to this function is not `options.CasADi_path = path`, it is `'CasADi_path', path`. So I'm not sure if it will be clear what `options` means to someone who calls this function without looking what's inside.

Reply:
Indeed, I understood wrongly that options is no struct.